### DISCUSSION TITLE
Support for Umbraco v9+

### DISCUSSION THREAD

**Discussion by karthik-phl** [Infocaster/UrlTracker#2] [Q&A] [answered]
Is there plans/roadmap to release a version of the Url tracker plugin for Umbraco v9? If so, is it possible for you to share timeframes?

Thanks.

**Comment by D-Inventor** [comment, marked as answer]
Hi there!

Good question. Yes, we are planning to port the URL Tracker to Umbraco v9. Here's what I can tell you: We're currently reworking the entire code base of the URL Tracker for Umbraco v8, this to make it easier to port to v9 afterwards, to improve performance, improve testability and to more easily accommodate future developments.

I cannot give any estimates or milestones unfortunately as we currently have our priorities elsewhere.

**Reply by karthik-phl** [reply]
Thanks for getting back. Glad to hear that the focus is on making the code base portable & robust. Is it reasonable to assume that Umbraco cloud will also be supported?

**Reply by D-Inventor** [reply]
We've never tried Umbraco Cloud to be honest, though I don't see any reason why it wouldn't work. It would be really awesome if anyone could give it a try and let us know how well it works.

So far, the only restriction is that the application should run with a SQL Server database, because one of the database queries can't be rephrased to work for SQL Compact Edition

**Comment by D-Inventor** [comment]
In case you're interested: We've released a prerelease of the URL Tracker version 5.0.0 with the new code base for Umbraco 8.
We're looking to gain some feedback to see if everything still works as intended, so if you're interested, you could give it a try!

**Reply by karthik-phl** [reply]
Thanks for the update, really appreciate it. Unfortunately, we're not on v8 yet, but planning to move to v8/9 sometime this year. Happy to try it out and give feedback at that time, though I suspect it'll be production ready by then :)
